types/gaugejs/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
@sandersn @orta do you think its ok to disable this rule or do we need something special here
The npm package name is not `gaugejs` https://www.npmjs.com/package/gaugeJS

Reply:
I don't know enough, will defer to Nathan 👍

Reply:
I started by naming the types directory using the exact name "gaugeJS" but then the DefinitelyTyped CI build errored out saying that the package name must be all lowercase.
There was no documentation on how to properly handle this case so I just tried different workarounds until I got the CI build passing.